homeassistant/components/nuki/lock.py

```diff
@@ -109,15 +115,25 @@ def device_state_attributes(self):
         """Return the device specific state attributes."""
         data = {
             ATTR_BATTERY_CRITICAL: self._battery_critical,
-            ATTR_NUKI_ID: self._nuki_lock.nuki_id}
+            ATTR_NUKI_ID: self._nuki_lock.nuki_id,
+            ATTR_AVAILABLE: self._available}
```

Comment:
Implement the `available` entity property instead.

Reply:
Done